src/browser_action/configuration.css

```
@@ -150,6 +150,19 @@ label[for="filter"] {
   font-weight: normal;
 }
 
+.script .note {
+  padding: 1ch 2ch;
+  margin: 0;
+
+  background-color: var(--in-content-box-info-background);
+  text-align: center;
+  font-size: 0.875rem;
```

since I'm already submitting a nit review now...
prefer misc styles to be alphabetised

```suggestion
  font-size: 0.875rem;
  text-align: center;
```